tests/Controller/ProductReviewApiTest.php

```diff
@@ -0,0 +1,359 @@
+        $response = $this->client->getResponse();
+
+        $this->assertResponse($response, 'product_review/accept_response', Response::HTTP_OK);
+    }
+
+    /**
+     * @test
+     */
+    public function it_does_not_allows_accept_product_review_if_it_has_not_new_status()
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yml');
+        $productReviewsData = $this->loadFixturesFromFile('resources/product_reviews.yml');
+
+        /** @var ProductInterface $product */
+        $product = $productReviewsData['product1'];
+
+         /** @var ProductReview $productReview */
+        $productReview = $productReviewsData['productReview2'];
+
+        $this->client->request('POST', $this->getReviewUrl($product, $productReview) . '/accept', [], [], static::$authorizedHeaderWithAccept);
+        $response = $this->client->getResponse();
+
+        $this->assertResponse($response, 'product_review/change_status_fail_response', Response::HTTP_BAD_REQUEST);
+    }
+
+    /**
+     * @test
+     */
+    public function it_allows_reject_product_review()
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yml');
+        $productReviewsData = $this->loadFixturesFromFile('resources/product_reviews.yml');
+
+        /** @var ProductInterface $product */
+        $product = $productReviewsData['product1'];
+
+         /** @var ProductReview $productReview */
+        $productReview = $productReviewsData['productReview1'];
+
+        $this->client->request('PATCH', $this->getReviewUrl($product, $productReview) . '/reject', [], [], static::$authorizedHeaderWithAccept);
+        $response = $this->client->getResponse();
+
+        $this->assertResponse($response, 'product_review/reject_response', Response::HTTP_OK);
+    }
+
+     /**
+     * @test
+     */
+    public function it_does_not_allows_reject_product_review_if_it_has_not_new_status()
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yml');
+        $productReviewsData = $this->loadFixturesFromFile('resources/product_reviews.yml');
+
+        /** @var ProductInterface $product */
+        $product = $productReviewsData['product1'];
+
+         /** @var ProductReview $productReview */
+        $productReview = $productReviewsData['productReview3'];
+
+        $this->client->request('POST', $this->getReviewUrl($product, $productReview) . '/accept', [], [], static::$authorizedHeaderWithAccept);
+        $response = $this->client->getResponse();
+
+        $this->assertResponse($response, 'product_review/change_status_fail_response', Response::HTTP_BAD_REQUEST);
+    }
+
+    /**
+     * @param ProductInterface $product
+     *
+     * @return string
+     */
+    private function getReviewListUrl(ProductInterface $product)
+    {
+        return sprintf('/api/v1/products/%s/reviews/', $product->getCode());
+    }
+
+    /**
+     * @param ProductInterface $product
+     * @param ProductReview $productReview
```

Comment:
`ProductReviewInterface`

Reply:
Hi, 
I tried to find the "ProductReviewInterface" but it does not exists, only "ReviewInterface". 
And because this class refers to the Product Review Api, I thought will be a good fit to use "ProductReview" model. Or I should create a interface for ProductReview model?

Reply:
My bad, there should be `ReviewInterface` instead of `ProductReview` 😃